modules/move-etcd-different-disk.adoc

```diff
@@ -10,20 +10,60 @@ You can move etcd from a shared disk to a separate disk to prevent or resolve pe
 
 The Machine Config Operator (MCO) is responsible for mounting a secondary disk for {product-title} {product-version} container storage.
 
+[NOTE]
+====
+This procedure only supports device names for the following device types:
+
+* SCSI or SATA- `/dev/sd*`
++
+* Virtual - `/dev/vd*`
++
+* NVMe - `/dev/nvme*[0-9]\*n*`
+====
+
+This script is for `/etc/find-secondary-device`. Decode and replace the device name in the script according to your environment.
```

Comment:
I don't think we need these sentences or the script in the introductory content since encoding the script is a step of the procedure. :slightly_smiling_face: